.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
```

Comment:
Sorry for the unrelated change here, but I had to re-initialize PhpStorm and realized a bunch of these settings need to be imported